### PR TITLE
Support for Previous and Next button labels

### PR DESCRIPTION
**Pull request by schoes**

### CODE REVIEW

File: dist/date-range-picker.js [hunk 1]
```diff
@@ -341,6 +343,8 @@
   function render(dp) {
     var opts = dp.opts;
     var lang = opts.lang;
+    var nextButton = lang.nextButton || 'Next';
+    var prevButton = lang.prevButton || 'Previuos';
```
Comment (chrisdavies) [inline dist/date-range-picker.js]: Typo: Previuos -> Previous

File: dist/date-range-picker.js [hunk 2]
```diff
@@ -352,14 +356,14 @@
     return (
       '<div class="dp-cal">' +
         '<header class="dp-cal-header">' +
-          '<button tabindex="-1" type="button" class="dp-prev">Prev</button>' +
+          '<button tabindex="-1" type="button" class="dp-prev">nextButton</button>' +
```
Comment (chrisdavies) [inline dist/date-range-picker.js]: This should be `' + nextButton + '`

File: dist/date-range-picker.js [hunk 3]
```diff
@@ -352,14 +356,14 @@
...
           '<button tabindex="-1" type="button" class="dp-cal-month">' +
             lang.months[hilightedMonth] +
           '</button>' +
           '<button tabindex="-1" type="button" class="dp-cal-year">' +
             hilightedDate.getFullYear() +
           '</button>' +
-          '<button tabindex="-1" type="button" class="dp-next">Next</button>' +
+          '<button tabindex="-1" type="button" class="dp-next">prevButton</button>' +
```
Comment (chrisdavies) [inline dist/date-range-picker.js]: This should be `' + prevButton + '`

File: dist/tiny-date-picker.js [hunk 4]
```diff
@@ -352,14 +356,14 @@
     return (
       '<div class="dp-cal">' +
         '<header class="dp-cal-header">' +
-          '<button tabindex="-1" type="button" class="dp-prev">Prev</button>' +
+          '<button tabindex="-1" type="button" class="dp-prev">nextButton</button>' +
```
Comment (chrisdavies) [inline dist/tiny-date-picker.js]: This should be `' + nextButton + '`

File: dist/tiny-date-picker.js [hunk 5]
```diff
@@ -352,14 +356,14 @@
...
           '<button tabindex="-1" type="button" class="dp-cal-month">' +
             lang.months[hilightedMonth] +
           '</button>' +
           '<button tabindex="-1" type="button" class="dp-cal-year">' +
             hilightedDate.getFullYear() +
           '</button>' +
-          '<button tabindex="-1" type="button" class="dp-next">Next</button>' +
+          '<button tabindex="-1" type="button" class="dp-next">prevButton</button>' +
```
Comment (chrisdavies) [inline dist/tiny-date-picker.js]: This should be `' + prevButton + '`

File: src/views/day-picker.js [hunk 6]
```diff
@@ -40,14 +42,14 @@ function render(dp) {
   return (
     '<div class="dp-cal">' +
       '<header class="dp-cal-header">' +
-        '<button tabindex="-1" type="button" class="dp-prev">Prev</button>' +
+        '<button tabindex="-1" type="button" class="dp-prev">nextButton</button>' +
```
Comment (chrisdavies) [inline src/views/day-picker.js]: This should be `' + nextButton + '`

File: src/views/day-picker.js [hunk 7]
```diff
@@ -40,14 +42,14 @@ function render(dp) {
...
         '<button tabindex="-1" type="button" class="dp-cal-month">' +
           lang.months[hilightedMonth] +
         '</button>' +
         '<button tabindex="-1" type="button" class="dp-cal-year">' +
           hilightedDate.getFullYear() +
         '</button>' +
-        '<button tabindex="-1" type="button" class="dp-next">Next</button>' +
+        '<button tabindex="-1" type="button" class="dp-next">prevButton</button>' +
```
Comment (chrisdavies) [inline src/views/day-picker.js]: This should be `' + prevButton + '`

**Comment by schoes** [comment]
hey Chris i made the changes you mentioned. can you please check it ?

**Pull request closed without merging**